R/render.R

```diff
@@ -116,7 +124,8 @@ moon_reader = function(
       if (identical(mathjax, 'default')) {
         mathjax = 'https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML'
       }
-      pandoc_args = c(pandoc_args, '-V', paste0('mathjax-url=', mathjax))
+      pandoc_args = c(pandoc_args, '-V', paste0('mathjax-url=', mathjax),
+                      "-V", paste0('title-slide-class=', titleSlideClass, ''))
```

Comment:
We need to move this out of the `if` clause (i.e. `title-slide-class` should not be conditional on mathjax).